frontend/src/components/AddRecipeForm.js

```diff
@@ -11,6 +11,28 @@ const showAlert = () => {
 };
 
 const AddRecipeForm = () => {
+  const [newCategory, setNewCategory] = useState('');
+  const [error, setError] = useState('');
+  async function addCategory(value) {
+    const data = {
+      name: value,
+    };
+
+    console.log(data);
+    data.name !== ''
+      ? setCategoryList([...categoryList, data])
+      : setCategoryList([...categoryList]);
```

Comment:
Miért seteljük újra, ha üres a name? És miért küldjük be a backendnek? Én már rögtön a függvény elején ellenőrizném a value-t és return, ha üres string.

Reply:
Done.